### DISCUSSION TITLE
support for binder (waydroid)

### DISCUSSION THREAD

**Discussion by slynobody** [xanmod/kernel#432] [General]
Is it possible to include / enable binder-support directly into the kernel (without patch)? applications like waydroid rely on it (what makes it difficult to run xmod on a daily basis). if not, why? if not, could you please write a how-to to enable the binder-patches?

**Comment by xanmod** [comment]
CONFIG_ANDROID_BINDER_IPC=m
CONFIG_ANDROID_BINDERFS=m
CONFIG_ANDROID_BINDER_DEVICES="binder,hwbinder,vndbinder"

Need add `psi=1` boot kernel parameter
https://wiki.archlinux.org/title/Waydroid#WARNING:_Service_manager_/dev/binder_has_died

**Comment by slynobody** [comment]
great, works. Thank you!
